crates/bevy_mikktspace/src/generated.rs

```diff
@@ -1808,3 +1809,63 @@ unsafe fn GenerateInitialVerticesIndexList<I: Geometry>(
     }
     return iTSpacesOffs;
 }
+
```

Comment:
Good idea to keep the cfg feature hell in one place. This does remind me of the functionality of `bevy_math::ops`, though.

Reply:
Yeah I think it would make sense to move this into `bevy_math`, but that's a much harder change since it includes (comparatively) lots of dependencies. For now I think this is an acceptable mess, but highlights the need for possible changes.